bots/machine/testvm_core.py

```diff
@@ -0,0 +1,557 @@
+#!/usr/bin/python3 -u
```

Comment:
If you really need to split this out, how about calling it ssh_connection.py, to match the exported class?

Reply:
I'll rename it